src/Microsoft.AspNet.SignalR.SqlServer/SqlMessageBus.cs

```diff
@@ -137,19 +137,24 @@ private void Initialize(object state)
         private void StartReceiving(int streamIndex)
         {
             var stream = _streams[streamIndex];
-            stream.StartReceiving()
-                // Open the stream once receiving has started
-                .Then(() => Open(streamIndex))
-                // Starting the receive loop failed
-                .Catch(ex =>
+
+            stream.StartReceiving().ContinueWith(async task =>
+            {
+                try
+                {
+                    await task;
+                    // Open the stream once receiving has started
+                    Open(streamIndex);
+                }
+                catch (Exception ex)
                 {
                     OnError(streamIndex, ex);
 
-                    // Try again in a little bit
+                    _logger.WriteWarning("Exception thrown by Task", ex);
```

Comment:
ScaleoutStream  and here are in different assembly and in different log category, probably it is still better to keep the log here.